CMakeLists.txt

```diff
@@ -817,6 +817,8 @@ if(NOT MSVC)
   append_cxx_flag_if_supported("-Wno-strict-aliasing" CMAKE_CXX_FLAGS)
   append_cxx_flag_if_supported("-Wvla-extension" CMAKE_CXX_FLAGS)
   append_cxx_flag_if_supported("-Wnewline-eof" CMAKE_CXX_FLAGS)
+  append_cxx_flag_if_supported("-Winconsistent-missing-override" CMAKE_CXX_FLAGS)
+  append_cxx_flag_if_supported("-Winconsistent-missing-destructor-override" CMAKE_CXX_FLAGS)
```

Comment:
Just curious: aren't those part of `-Wall`?

Reply:
No, as you can see from clang doc

Reply:
If you ever plan to do the followup PR, mind adding a comments explaining why those  must be added on top of `-Wall` and `-Wextra` flags (as I've assumed those two should be all inclusive)